packages/codegen-ui/lib/types/form/index.ts

```diff
@@ -52,6 +53,8 @@ export type StudioForm = {
   sectionalElements: { [elementName: string]: SectionalElement };
 
   style: StudioFormStyle;
+
+  ctaConfig?: StudioFormButtons;
```

Comment:
Can we change the name of this type to match the property name better? Maybe StudioFormCTAConfig

Reply:
updated